scripts/changelog_check.rb

```diff
@@ -99,7 +112,7 @@ def generate_changelog(base_branch, source_branch)
       category: category,
       subcategory: change[:subcategory].capitalize,
       pr_number: pr_number&.named_captures&.fetch('pr'),
-      change: change[:change].capitalize,
+      change: change[:change],
```

Comment:
Per [Slack discussion](https://gsa-tts.slack.com/archives/C0NGESUN5/p1648042618648259?thread_ts=1647976406.207979&cid=C0NGESUN5) and [related blog post](https://glaucocustodio.github.io/2016/05/19/rails-5-new-upcase-first-method/), maybe this?

```suggestion
      change: change[:change].sub(/./, &:upcase),
```